src/tauri/src/lib/pages/playlunky/ModListItemDragHandle.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { buttonStyles } from "../../common/Button.svelte"
+
+  export let styleClass = "";
+</script>
+
+<div
+  {...$$props}
+  on:mousedown
+  on:mouseup
+  on:touchstart
+  on:touchend
+  class="{buttonStyles.base}
+    {buttonStyles.size['default']}
```

Comment:
I mildly prefer dot-notation (`buttonStyles.size.default`) over bracket notation. (Same applies to `transparent` below. WDYT?

Reply:
ah true it should be better like how you said, I didn't really think about that, I just copied that part of code from the button component 😅

Reply:
Should I squash the commit? idk how it works after already pushing the changes